esx_service/utils/auth_api.py

```diff
@@ -67,7 +67,7 @@ def get_tenant_from_db(name):
 
     error_msg, tenant = auth_mgr.get_tenant(name)
     if error_msg:
-        error_info = generate_error_info_for_internal_error(error_msg)
+        error_info = error_info.generate_error_info(error_msg)
```

Comment:
I don't think this will compile - it should be error_code.xxx instead of error_info.xxx